docs/ai-consumers/service.md

```diff
@@ -5,9 +5,24 @@ keywords: intro concepts
 comments: false
 
 # Hero section
-title: AI Consumers
+title: AI Developers
```

Comment:
lets fill in the title based on the Folder the document is present ?

Reply:
Document is under ai-consumers

Reply:
Title is now redundant. I will remove this. I pick up title from the path directly